packages/patternfly-4/react-core/src/experimental/components/OverflowMenu/OverflowMenu.md

```diff
@@ -0,0 +1,303 @@
+            <OverflowMenuContentItem>Item</OverflowMenuContentItem>
+            <OverflowMenuContentItem>Item</OverflowMenuContentItem>
+          </OverflowMenuContentGroup>
+        </OverflowMenuContent>
+        <OverflowMenuControl>
+          <Dropdown
+            onSelect={this.onSelect}
+            toggle={<KebabToggle onToggle={this.onToggle} />}
+            isOpen={isOpen}
+            isPlain
+            dropdownItems={dropdownItems}
+          />
+        </OverflowMenuControl>
+      </OverflowMenu>
+    )
+  }
+}
+```
+
+## Overflow menu group types
+```js
+import * as React from 'react';
+import { OverflowMenu, OverflowMenuControl, OverflowMenuContent, OverflowMenuContentGroup, OverflowMenuContentItem, OverflowMenuDropdownItem } from '@patternfly/react-core/dist/esm/experimental';
+import { Dropdown, KebabToggle, Button } from '@patternfly/react-core';
+
+class OverflowMenuGroupTypes extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isOpen: false
+    };
+    this.onToggle = isOpen => {
+      this.setState({
+        isOpen
+      });
+    };
+    this.onSelect = event => {
+      this.setState({
+        isOpen: !this.state.isOpen
+      });
+    };
+  }
+
+  render() {
+    const { isOpen } = this.state;
+    const dropdownItems = [
+      <OverflowMenuDropdownItem key="action" isShared>Action</OverflowMenuDropdownItem>,
+      <OverflowMenuDropdownItem key="item1" isShared>Item 1</OverflowMenuDropdownItem>,
+      <OverflowMenuDropdownItem key="item2" isShared>Item 2</OverflowMenuDropdownItem>,
+      <OverflowMenuDropdownItem key="item3" isShared>Item 3</OverflowMenuDropdownItem>,
+      <OverflowMenuDropdownItem key="item4" isShared>Item 4</OverflowMenuDropdownItem>,
+      <OverflowMenuDropdownItem key="item5" isShared>Item 5</OverflowMenuDropdownItem>,
+    ];
+    return (
+      <OverflowMenu breakpoint="lg">
+        <OverflowMenuContent>
+          <OverflowMenuContentGroup> 
+            <OverflowMenuContentItem>Item</OverflowMenuContentItem>
+            <OverflowMenuContentItem>Item</OverflowMenuContentItem>
+            <OverflowMenuContentItem>Item</OverflowMenuContentItem>
+          </OverflowMenuContentGroup>
+          <OverflowMenuContentGroup groupType="button">
+            <OverflowMenuContentItem>
+              <Button variant="primary">Primary</Button>
+            </OverflowMenuContentItem>
+            <OverflowMenuContentItem>
+              <Button variant="secondary">Secondary</Button>
+            </OverflowMenuContentItem>
+            <OverflowMenuContentItem>
+              <Button variant="tertiary">Tertiary</Button>
+            </OverflowMenuContentItem>
+          </OverflowMenuContentGroup>
+          <OverflowMenuContentGroup groupType="icon">
+            <OverflowMenuContentItem>
+              <Button variant="link" aria-label="Align left">
+                <i className="fas fa-align-left" aria-hidden="true"></i>
```

Comment:
should you be using the react-icons package instead of font awesome? I don't typically see react components using font awesome.

Reply:
Updated - thanks for the heads up!